policyengine_uk/parameters/gov/dfe/targeted_childcare_entitlement/max_income_tc_recipients.yaml

```
@@ -0,0 +1,12 @@
+description: Families claiming the Tax Credits with income over this amount are ineligible for the benefit-targeted free childcare entitlement.
```

Is this true? Can they still receive it if they qualify via other benefits?

---

Yes, they receive if they qualify via other benefits. The gov website says:

_Your 2-year-old can get free childcare if you live in England and get any of the following benefits:

- Income Support
- income-based Jobseeker’s Allowance (JSA)
- income-related Employment and Support Allowance (ESA)
- Universal Credit, and your household income is £15,400 a year or less after tax, not including benefit payments
the guaranteed element of Pension Credit
- Child Tax Credit, Working Tax Credit (or both), and your household income is £16,190 a year or less before tax
the Working Tax Credit 4-week run on (the payment you get when you stop qualifying for Working Tax Credit)_

---

Right, the description indicates they are not eligible via other pathways if their income exceeds the limit. This fixes.
```suggestion
description: The Department for Education qualifies recipients of the Working Tax Credit or Child Tax Credit for targeted childcare entitlement if their income does not exceed this threshold.
```